tests/resources/helper_functions.py

```diff
@@ -257,7 +257,7 @@ def deploy_erc721_erc20(
             erc721_publisher.address,
             ZERO_ADDRESS,
         ],
-        uints=[web3.toWei("0.5", "ether"), 0],
+        uints=[cap, 0],
```

Comment:
Cap argument added as it was accidentally removed here: https://github.com/oceanprotocol/ocean.py/commit/f28d07c0300f2f4810f193708531e78004eab0c1